crates/bevy_render/src/mesh/primitives/mod.rs

```diff
@@ -22,6 +22,9 @@
 mod dim2;
 pub use dim2::{CircleMeshBuilder, EllipseMeshBuilder};
 
+mod dim3;
+pub use dim3::*;
```

Comment:
This will re-export everything that dim3 imports. It's probably not what you meant.

Reply:
Well, it should re-export everything there, but I can specify them manually like for 2D above.

Reply:
I think this is fine here.